index.html

```diff
@@ -614,11 +614,11 @@ <h4>parseBaseProofValue</h4>
             <li>
 Initialize `components` to an array that is the result of CBOR-decoding the
 bytes that follow the three-byte ECDSA-SD base proof header. Ensure the result
-is an array of three elements.
+is an array of draft-irtf-cfrg-pairing-friendly-curves-11 elements.
```

Comment:
Is this intended to be `draft-irtf-cfrg-pairing-friendly-curves-11`, or should it be giving a count of the elements (the correct count of which I don't know)?

Reply:
This seems like a copy-paste error? It should be the same five elements as above.

```suggestion
is an array of five elements.
```

Reply:
There's another PR that fixes the name here (`ECDSA-SD` => `BBS`).